### DISCUSSION TITLE
Add support for local shortcuts

### DISCUSSION THREAD

**Discussion by curtisblackwell** [NativePHP/laravel#214] [Ideas]
There's already support for global shortcuts, but not [local shortcuts](https://www.electronjs.org/docs/latest/tutorial/keyboard-shortcuts#local-shortcuts).

Would you merge a PR adding support for local shortcuts that follows the contribution guide?

**Comment by mpociot** [comment]
Yeah sure!

**Reply by curtisblackwell** [reply]
I was struggling with getting tests to pass in the NativePHP/electron-plugin and realized they were failing in [the latest merge](https://github.com/NativePHP/electron-plugin/actions/runs/6261874829/job/17002845671). Should I ignore tests in that repo for now?

**Comment by curtisblackwell** [comment]
I discovered I was incorrect. Local shortcuts are supported, [they're just called hotkeys in the docs and they're on the Application Menu page](https://nativephp.com/docs/1/the-basics/application-menu#hotkeys). This makes sense since, as far as I can tell, Electron only supports local shortcuts connected to menu items.
